src/scripts/visualizations/tooltip.js

```diff
@@ -47,6 +47,7 @@
             var pointX = xScale ? xScale(d.x) : pointOrCentroid.call(this)[0];
             var pointY = yScale ? yScale(d.y) : pointOrCentroid.call(this)[1];
             var alignedRight;
+            var followCursor = this.options.tooltip.followCursor;
```

Comment:
can you add an entry for `followCursor` under the 'defaults' in the beginning of this file? Listing it there is useful to quickly scan and get a comprehensive list of available options.